code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -149,14 +149,14 @@
 					return
 			if(isnull(construction_stage) || !reinf_material)
 				to_chat(user, SPAN_NOTICE("You begin removing the outer plating..."))
-				if(I.use_tool(user, src, WORKTIME_LONG, tool_type, FAILCHANCE_NORMAL, required_stat = STAT_MEC))
+				if(I.use_tool(user, src, WORKTIME_SLOW * material.heat_resistance, tool_type, FAILCHANCE_NORMAL, required_stat = STAT_MEC))
```

Comment:
Um-m... What  are theoretical welding times for materials?

Reply:
Base level: 12 seconds, modified by Mechanical stat.
Cloth is 3s, wood 0.5x, Plasteel is 3x, Titanium 4x, Sandstone and all children 8x, Osmium 10x, and if cardboard could be a wall, it would also have 3s.